apps/wallet-mobile/src/features/Swap/common/helpers.ts

```diff
@@ -95,3 +97,30 @@ export const sortTokensByName = (a: Balance.TokenInfo, b: Balance.TokenInfo, wal
 
   return nameA.localeCompare(nameB, undefined, {sensitivity: 'base'})
 }
+
+export const getPriceImpactStatus = (priceImpact: number) => {
+  if (priceImpact < 1) return 'positive'
+  if (priceImpact > 10) return 'negative'
+  return 'warning'
+}
+
+export const usePriceImpactStatusTheme = (status: SwapPriceImpactStatus) => {
+  const {theme} = useTheme()
+
+  if (status === 'negative') {
+    return {
+      text: theme.color.magenta[500],
+      background: theme.color.magenta[100],
+    }
+  } else if (status === 'warning') {
+    return {
+      text: theme.color.yellow[500],
+      background: theme.color.yellow[100],
+    }
+  }
+
+  return {
```

Comment:
this should return the default colours for `none` - black (even if its not on the screen for now) when is `none`